packages/phrases/src/locales/de/translation/admin-console/application-details.ts

```diff
@@ -41,6 +41,9 @@ const application_details = {
   enable_admin_access: 'Admin-Zugang aktivieren',
   enable_admin_access_label:
     'Zugang zur Management API aktivieren oder deaktivieren. Wenn aktiviert, können Access Tokens verwendet werden, um die Management API im Namen der Anwendung aufzurufen.',
+  always_issue_refresh_token: 'Always issue Refresh Token',
```

Comment:
nit: why is this line not translated. lol
```suggestion
  always_issue_refresh_token: 'Geben Sie immer ein Aktualisierungstoken aus',
```